MetaMask/metamask-extension
# UX: Multichain: Fix AccountPicker Padding when Address Showing

## **Description**
The `ButtonBase` component has a default size of `Md`.  Since we recently started showing the address within the `AccountPicker`, the contents have gotten taller which has eliminated some padding.  This PR adds more vertical padding when the address is displaying.

## **Manual testing steps**

1.  See no change on regular `yarn start`
2. Start the extension with `MULTICHAIN=1 yarn start`
3. See more padding on the `AccountPicker`

## **Screenshots/Recordings**

### **Before**

<img width="658" alt="SCR-20231008-rzmt" src="https://github.com/MetaMask/metamask-extension/assets/46655/210b86c5-00e5-431e-9a3c-1fd44f9270b2">

### **After**

<img width="923" alt="SCR-20231008-rzuq" src="https://github.com/MetaMask/metamask-extension/assets/46655/ac31928c-d36a-4851-a598-5c31959ec4a7">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
